README.md

```diff
@@ -12,35 +12,35 @@
-  - [Automatically finds tsconfig.json](#automatically-finds-tsconfigjson)
-  - [Supports synthetic modules](#supports-synthetic-modules)
-  - [Supports automatic of jest.mock() calls](#supports-automatic-of-jestmock-calls)
-- [Configuration](#configuration)
-  - [tsconfig](#tsconfig)
-  - [Module path mapping](#module-path-mapping)
-  - [Skipping Babel](#skipping-babel)
-  - [Using `.babelrc`](#using-babelrc)
-  - [Using a custom Babel config](#using-a-custom-babel-config)
-  - [TS compiler & error reporting](#ts-compiler--error-reporting)
-  - [Ignore coverage on decorators](#ignore-coverage-on-decorators)
-- [Use cases](#use-cases)
-  - [React Native](#react-native)
-- [Angular 2](#angular-2)
-- [Using ES2015+ features in Javascript files](#using-es2015-features-in-javascript-files)
-- [Tips](#tips)
-  - [Importing packages written in TypeScript](#importing-packages-written-in-typescript)
-- [Known Limitations](#known-limitations)
-  - [Known limitations for TS compiler options](#known-limitations-for-ts-compiler-options)
-  - [Known Limitations for hoisting](#known-limitations-for-hoisting)
-  - [`const enum` is not supported](#const-enum-is-not-supported)
-- [How to Contribute](#how-to-contribute)
-  - [Quickstart to run tests (only if you're working on this package)](#quickstart-to-run-tests-only-if-youre-working-on-this-package)
-- [License](#license)
+  - [Usage](#usage)
+    - [Versioning](#versioning)
+    - [Coverage](#coverage)
+  - [Default Setup](#default-setup)
+    - [Sourcemap support](#sourcemap-support)
+    - [Automatically finds tsconfig.json](#automatically-finds-tsconfigjson)
+    - [Supports synthetic modules](#supports-synthetic-modules)
+    - [Supports automatic of jest.mock() calls](#supports-automatic-of-jestmock-calls)
+  - [Configuration](#configuration)
+    - [tsconfig](#tsconfig)
+    - [Module path mapping](#module-path-mapping)
+    - [Skipping Babel](#skipping-babel)
+    - [Using `.babelrc`](#using-babelrc)
+    - [Using a custom Babel config](#using-a-custom-babel-config)
+    - [TS compiler & error reporting](#ts-compiler--error-reporting)
+    - [Ignore coverage on decorators](#ignore-coverage-on-decorators)
+  - [Use cases](#use-cases)
+    - [React Native](#react-native)
+  - [Angular 2](#angular-2)
+  - [Using ES2015+ features in Javascript files](#using-es2015-features-in-javascript-files)
+  - [Tips](#tips)
+    - [Importing packages written in TypeScript](#importing-packages-written-in-typescript)
+  - [Known Limitations](#known-limitations)
+    - [Known limitations for TS compiler options](#known-limitations-for-ts-compiler-options)
+    - [Known Limitations for hoisting](#known-limitations-for-hoisting)
+    - [`const enum` is not supported](#const-enum-is-not-supported)
+  - [How to Contribute](#how-to-contribute)
+    - [Quickstart to run tests (only if you're working on this package)](#quickstart-to-run-tests-only-if-youre-working-on-this-package)
+  - [License](#license)
```

Comment:
Can you actually undo the spaces change here please?

Reply:
Done, it's my markDown plugin to update the tableOfContent on save...
Sorry about this.